recipes/xcube/meta.yaml

```diff
@@ -36,11 +35,12 @@ requirements:
     - numpy >=1.16
     - pandas >=0.24
     - pillow >=6.0
-    - proj >=5.2
+    - proj4 >=5.2
```

Comment:
If you do not depend on proj directly let `gdal` pull the right proj and avoid naming and version issues with a pin like that.

Reply:
Removed proj for the next push